src-ui/lib/db/index.ts

```diff
@@ -0,0 +1,28 @@
+import { drizzle } from 'drizzle-orm/sqlite-proxy'
+import { commands } from '$lib/generated/sqlite_proxy'
+import * as schema from '$lib/drizzle/schema'
+
+export const db = drizzle(
```

Comment:
Following https://orm.drizzle.team/docs/connect-drizzle-proxy and connecting to our shiney backend sqlite proxy.